packages/geoprocessing/src/toolbox/overlapRaster.ts

```diff
@@ -39,7 +41,7 @@ export async function overlapRaster(
       createMetric({
         metricId,
         sketchId: sumFeatures[index].properties.id,
-        value: curSum,
+        value: simplifyPrecision ? parseFloat(curSum.toPrecision(6)) : curSum,
```

Comment:
check out the `roundDecimal` function in `helpers/number.ts` that may handle the truncating a little better

Reply:
Oh cool, I didn't see that function. Cool, I'll have it truncate to 6 decimal digits.

Reply:
Double checking this will return the values you expect, the way `roundDecimal(value, decimals)` is currently working will return these numbers:
`roundDecimal(1, 6)` -> 1
`roundDecimal(0.000000858374, 6)` -> 0
`roundDecimal(8.58e-7, 6)` -> 0
`roundDecimal(1111.1111119, 6)` -> 1111.111112
So rounding to 6 digits after the decimal point. Checking because some of our ous values end up being e-7 or tinier and those will get truncated to 0.

Reply:
Yes that is what I would expect.  I'm open to feedback.

Reply:
Okay, great. That's fine just wanted to check